wesnoth-did.po

```diff
@@ -6344,45 +6365,55 @@ msgid ""
 "This attack loses one strike on offense but deals three times the normal "
 "damage."
 msgstr ""
+"Počet útoků se při ofenzivním použití snižuje o jeden, ale každý útok "
+"způsobuje trojnásobné zranění."
 
 #. [attacks]: id=did_corpse_launcher
 #: data/campaigns/Descent_Into_Darkness/utils/abilities.cfg:541
 msgid "corpse launcher"
-msgstr ""
+msgstr "vymršťovač mrtvol"
 
+# Asi bych ten útok potřeboval vidět v praxi.
 #. [attacks]: id=did_corpse_launcher
 #: data/campaigns/Descent_Into_Darkness/utils/abilities.cfg:542
+#, fuzzy
 msgid ""
 "On offense, enemies cannot retaliate against this attack. This attack "
 "consumes 3 HP from each nearby allied corpse to deal 11 impact damage (per "
 "corpse) to the target on top of this attack’s base damage."
 msgstr ""
+"Při ofenzivním použití se nemohou nepřátelé proti tomuto útoku bránit "
+"protiútokem. Z každé sousedící spřátelené chodící mrtvoly se vytáhnou 3 "
+"životy, které se zformují do útoku úderem v síle 11 (za každou mrtvolu) do "
+"cíle na horní hranici základního poškození tohoto útoku."
```

Comment:
```suggestion
"životy, které se zformují do útoku na cíl úderem v síle 11 (za každou mrtvolu), "
"jenž se přidá k základnímu poškození tohoto útoku."
```

Reply:
No to mi právě přišlo jako hrozně velký úder. Ale překlad je správně, tak proč to zatím nenechat.